src/SourceLocator/SourceLocator.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Asgrim\SourceLocator;
+
+interface SourceLocator
```

Comment:
@asgrim other symbols that can be defined are:
 - constants
 - traits
 - interfaces

Not sure if a single locator has to deal with all of this or not